src/Analyzers/Core/CodeFixes/RemoveUnusedMembers/AbstractRemoveUnusedMembersCodeFixProvider.cs

```diff
@@ -123,10 +123,10 @@ protected static void AdjustAndAddAppropriateDeclaratorsToRemove(SyntaxNode pare
             }
         }
 
-        private class MyCodeAction : CodeAction.DocumentChangeAction
+        private class MyCodeAction : CustomCodeActions.DocumentChangeAction
```

Comment:
Does this mean telemetry will continue to work?  Or will this explicitly break it?

Reply:
Telemetry should have fully qualified type name of the code action - so as long as we are using special nested types for code actions, we should generate telemetry. I will still look around in code and confirm.

Reply:
Yes, we should still get telemetry: https://github.com/dotnet/roslyn/blob/ff9cc22cddbe04895fdcc1be5f4b1552da0648ea/src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/SuggestedAction.cs#L67-L71